### PR TITLE
Output curl to file

### PR DESCRIPTION
**Pull request by fxiang1**
https://issues.redhat.com/browse/ACM-10683

Curl is writing to stdout, need to write to file instead.

- Output curl to file

### CODE REVIEW

**Review by mikeshng — APPROVED** [review]
/approve

/lgtm

File: ci-operator/config/stolostron/cluster-curator-controller/stolostron-cluster-curator-controller-main.yaml [hunk 1]
```diff
@@ -37,7 +37,7 @@ tests:
     export os=$(go env GOOS)
     export arch=$(go env GOARCH)
     export KB_TOOLS_ARCHIVE_NAME="kubebuilder-tools-${KUBEBUILDER_VERSION}-${os}-${arch}.tar.gz"
-    curl -s -f -L https://storage.googleapis.com/kubebuilder-tools/${KB_TOOLS_ARCHIVE_NAME}
+    curl -s -f -L https://storage.googleapis.com/kubebuilder-tools/${KB_TOOLS_ARCHIVE_NAME} -o ${KB_TOOLS_ARCHIVE_NAME}
     tar -zxvf ${KB_TOOLS_ARCHIVE_NAME}
     sudo mv -f ./kubebuilder /usr/local/kubebuilder
```
Comment (mikeshng) [inline ci-operator/config/stolostron/cluster-curator-controller/stolostron-cluster-curator-controller-main.yaml:42]: this sudo usually prompt for a password. your best bet is just to use mv without sudo if this doesn't work

**Review by mikeshng — APPROVED** [review]

File: ci-operator/config/stolostron/cluster-curator-controller/stolostron-cluster-curator-controller-main.yaml (hunk 1 above)
Comment (fxiang1) [reply ci-operator/config/stolostron/cluster-curator-controller/stolostron-cluster-curator-controller-main.yaml:42]: Good point Mike. The other issue could be a read-only file system in which case I'm tempted to use kubebuilder from /tmp. Thanks!

Comment (xiangjingli) [reply ci-operator/config/stolostron/cluster-curator-controller/stolostron-cluster-curator-controller-main.yaml:42]: @fxiang1  may try to set the PATH to ./kubebuilder before running sonar cloud and reset PATH to original after sonar cloud scan.

Comment (fxiang1) [reply ci-operator/config/stolostron/cluster-curator-controller/stolostron-cluster-curator-controller-main.yaml:42]: Thanks @xiangjingli I'll try to set the PATH and also KUBEBUILDER_ASSETS which overwrites the default kubebuilder path.

**Pull request merged**